vector/src/main/res/values/donottranslate.xml

```diff
@@ -28,4 +28,7 @@
     <string name="ftue_auth_choose_server_ems_subtitle">Element Matrix Services (EMS) is a robust and reliable hosting service for fast, secure and real time communication. Find out how on <a href="${ftue_ems_url}">element.io/ems</a></string>
     <string name="ftue_auth_choose_server_ems_cta">Get in touch</string>
 
+    <string name="ftue_auth_terms_title">Our privacy policy</string>
```

Comment:
`Our` may be confusing for some user. Is it referring to 'Element' app or to the homeserver?

Previous wording was more explicit and better IMO:

<img width="348" alt="image" src="https://user-images.githubusercontent.com/3940906/163210740-e648d903-5c5a-4d3c-a52a-152cc877f890.png">

Reply:
will raise in the FTUE room :+1: 

cc @ericdecanini the strings are in `donottranslate.xml` which is not synced with weblate whilst the feature is under development (for these cases where we should iterate!)

Reply:
will update to just `Privacy policy`

Reply:
LGTM!